README.md

```diff
@@ -51,40 +73,92 @@ You can then execute your binary: `./target/open-management-portal-git-api-1.0.0
 
 If you want to learn more about building native executables, please consult https://quarkus.io/guides/building-native-image-guide .
 
-## Add basic JDG to OCP
+## Add Infinispan to OCP via the Infinispan Operator
+
+* Cluster admin probably needed for CRD
+
+### Create secret
+
+Create a secret file named `omp-cache-secret.yaml`. This secret will create an `identities.yaml` file on the cache. Take note of the name that must match the app secret By default the user / pass will be omp / omp as defined in `src/main/resources/application.properties` and the secret below.
+
+```
+apiVersion: v1
+data:
+  identities.yaml: Y3JlZGVudGlhbHM6Ci0gdXNlcm5hbWU6IG9tcAogIHBhc3N3b3JkOiBvbXAKLSB1c2VybmFtZTogb3BlcmF0b3IKICBwYXNzd29yZDogTDdFYXZOQklBRXhORDVHdAo=
```

Comment:
Just double checking - the "operator" password in here is not a legit password for anything, right?

Reply:
it matches the default creds in the app properties

Reply:
@jacobsee I update this to reflect that this example matches the defaults by using plain text instead

Reply:
👍